internal/cmd/create.go

```diff
@@ -0,0 +1,244 @@
+		Short: fmt.Sprintf("%s create: creates a new VEX document", appname),
+		Long: fmt.Sprintf(`%s create: creates a new VEX document
+
+The create subcommand generates a single statement document
+from the command line. This is intended for simple use cases
+or to get a base document to get started.
+
+You can specify multiple products and customize the metadata of
+the document via the command line flags. vexctl will honor the
+SOURCE_DATE_EPOCH environment variable and use that date for 
+the document (it can be formated in unix time or RFC3339).
+
+If you don't specify an ID for the document, one will be generated
+using its canonicalization hash.
+
+Examples:
+
+# Generate a document stating that CVE-2023-12345 was fixed in the 
+# git package of Wolfi:
+
+%s create "pkg:apk/wolfi/git@2.39.0-r1?arch=x86_64" CVE-2023-12345 fixed
+
+# You can specify more than one product. vexctl will read one from
+# the argument but you can control all parameters through command line
+# flags. Here's an example with two products in the same document:
+
+%s create --product="pkg:apk/wolfi/git@2.39.0-r1?arch=x86_64" \
+              --product="pkg:apk/wolfi/git@2.39.0-r1?arch=armv7" \
+              --vuln="CVE-2023-12345" \
+              --status="fixed"
+
+# not_affected statements need a justification:
+
+%s create --product="pkg:apk/wolfi/trivy@0.36.1-r0?arch=x86_64" \
+              --vuln="CVE-2023-12345" \
+              --status="not_affected" \
+              --justification="component_not_present" 
+
+`, appname, appname, appname, appname),
+		Use:               "create [flags] [product_id [vuln_id [status]]]",
+		Example:           fmt.Sprintf("%s create \"pkg:apk/wolfi/git@2.39.0-r1?arch=x86_64\" CVE-2022-39260 fixed ", appname),
+		SilenceUsage:      false,
+		SilenceErrors:     false,
+		PersistentPreRunE: initLogging,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := opts.Validate(args); err != nil {
+				return err
+			}
+			// If we have arguments, add them
+			for i := range args {
+				switch i {
+				case 0:
+					opts.Products = append(opts.Products, args[i])
+				case 1:
+					opts.Vulnerability = args[i]
+				case 2:
+					opts.Status = args[i]
+				}
+			}
+			newDoc := vex.New()
+
+			statement := vex.Statement{
+				Vulnerability:   opts.Vulnerability,
+				Products:        opts.Products,
+				Subcomponents:   opts.Subcomponents,
+				Status:          vex.Status(opts.Status),
+				StatusNotes:     opts.StatusNotes,
+				Justification:   vex.Justification(opts.Justification),
+				ImpactStatement: opts.ImpactStatement,
+			}
+			newDoc.Statements = append(newDoc.Statements, statement)
+			if _, err := newDoc.GenerateCanonicalID(); err != nil {
+				return fmt.Errorf("generating document id: %w", err)
+			}
+
+			out := os.Stdout
+
+			if opts.outFilePath != "" {
+				f, err := os.Create(opts.outFilePath)
+				if err != nil {
+					return fmt.Errorf("opening vex file to write document: %w", err)
```

Comment:
nit: I noticed sometimes we say "vex" and sometimes we say "VEX". Does that matter?

Reply:
I've changed them all to "VEX"